### PR TITLE
Add Statistics line to log and GVRConsole

### PR DESCRIPTION
**Pull request by danke-sra**

### CODE REVIEW

**Comment by danke-sra** [comment]
Can this be merged for use? There is a flag that can disable it when needed.

File: GVRf/Framework/src/org/gearvrf/GVRViewManager.java [hunk 1]
```diff
@@ -83,6 +86,8 @@
 class GVRViewManager extends GVRContext implements RotationSensorListener {
 
     private static final String TAG = Log.tag(GVRViewManager.class);
+    private static final boolean DEBUG_STATS = true;
```
Comment (liaxim) [inline GVRf/Framework/src/org/gearvrf/GVRViewManager.java]: Should this be false by default?

**Comment by liaxim** [comment]
Squash maybe?

**Comment by danke-sra** [comment]
No problem.

File: GVRf/Framework/src/org/gearvrf/GVRViewManager.java (hunk 1 above)
Comment (jason2kim) [inline GVRf/Framework/src/org/gearvrf/GVRViewManager.java]: Let's keep all debug flags off even if overhead is small. We can turn if on whenever we need to collect it.
    protected static final boolean DEBUG_STATS = false; // keep it off

**Comment by danke-sra** [comment]
Updated.

**Pull request merged**